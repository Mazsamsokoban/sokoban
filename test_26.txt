loadtest 26
showstate f1
put w honey
showstate f1