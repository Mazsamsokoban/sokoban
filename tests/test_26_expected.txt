f1:w 

name:f1
friction:Normal
thing:w

name:f1
friction:Honey
thing:w

